reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -613,11 +613,13 @@ <h3 class="">Packages</h3>
             <td><a href="#Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0-pkg-1">1</a></td><td>Gradle:org.ossreviewtoolkit.gradle.example:lib:1.0.0</td><td></td><td><em>Detected Licenses (from <a href="https://github.com/oss-review-toolkit/ort.git">VCS</a>):</em>
               <dl>
                 <dd>
-                  <div>BSD-3-Clause (<a href="https://github.com/oss-review-toolkit/ort/tree/master/analyzer/src/funTest/assets/projects/synthetic/gradle/lib/src/code.cpp#L1">link</a> to the location)</div>
+                  <div>
```

Comment:
Commit message:

> SoftwareHeritage

"Software Heritage"

> consistent link

"consistent links independent of the source code hoster"

> the software heritage links were used

"links to Software Heritage are used"

> Using a link to the license files in the ORT repository might not be consistent across forks or version.

This sentence comes a bit out of the blue, as you didn't explain before that this was considered as an option. Maybe just drop it?